README.md

```diff
@@ -34,6 +34,10 @@ The API and models are actually defined in Java with both `swagger` and `jax-rs`
 
 Note that the build actually generates these files under the `src/main/resources/generated` folder, but this folder is not committed on the git repository. The `maven release` process actually copies the generated files to the folders mentioned above so that we only have released files in the repository.
 
+### GraphQL endpoint and specification
```

Comment:
Since the headline is endpoint and specification we also should mention `/graphql` endpoint in one sentence.

Reply:
I slightly changed the text but I don't really find it confusing. We have `/graphql` at the REST level, and if you want to find out about all the GraphQL schema and what it can do, there is the other doc.

Reply:
LGTM now.